delft/textClassification/models.py

```diff
@@ -1,31 +1,19 @@
-import numpy as np
-import sys, os
-import argparse
-import math
 import json
-import time
-import shutil
-
-from delft.textClassification.data_generator import DataGenerator
-
-from tensorflow.keras import initializers, regularizers, constraints
-from tensorflow.keras.models import Model, load_model
-from tensorflow.keras.layers import Dense, Embedding, Input, InputLayer, concatenate
-from tensorflow.keras.layers import LSTM, Bidirectional, Dropout, SpatialDropout1D, AveragePooling1D, GlobalAveragePooling1D, TimeDistributed, Masking, Lambda 
-from tensorflow.keras.layers import GRU, MaxPooling1D, Conv1D, GlobalMaxPool1D, Activation, Add, Flatten, BatchNormalization
-from tensorflow.keras.losses import SparseCategoricalCrossentropy
-from tensorflow.keras.optimizers import RMSprop, Adam, Nadam, schedules
-from tensorflow.keras.preprocessing import text, sequence
-from tensorflow.keras.callbacks import EarlyStopping, ModelCheckpoint
-from tensorflow import keras
+import math
```

Comment:
Overall this class is not updated correct? It does not use the new transformer wrapper, still use `config.transformer` as before, etc. and direct AutoConfig call without local loading. I see only cleaning/renaming changes.